### DISCUSSION TITLE
Installing ActiveAdmin v4 without cssbundling with Tailwind Play CDN

### DISCUSSION THREAD

**Discussion by matteeyah** [activeadmin/activeadmin#8273] [Help]
## Summary

Hey, did anyone manage to install ActiveAdmin v4 without `cssbundling-rails`? Our project doesn't use any `node` packages, and we won't change that for ActiveAdmin, but we'd still love if we could install the new version of ActiveAdmin.

So far, I've managed to overwrite the head partial, and included the Tailwind Play CDN with

```
<%= javascript_include_tag "https://cdn.tailwindcss.com" %>
```

It seems to be working, but things still look pretty off. Also, no `data-` HTML attributes seem to work, hence all buttons that rely on them also don't work. Does anyone have an idea what it might be related to?

<img width="1552" alt="Screenshot 2024-03-01 at 15 49 09" src="https://github.com/activeadmin/activeadmin/assets/1218868/863ed827-bbc0-4500-b745-64a1caa7c07c">

There's no errors in the JavaScript console, and the generated import map seems fine. I can load all of the individual imports in the Network tab of developer tools, so all of those JS pieces are present. Could it be that the Tailwind Play CDN can't be used like this?

**Comment by mgrunberg** [comment]
@matteeyah you can try using `tailwind-rails` gem. There is no official way to replace `cssbundling-rails` with `tailwind-rails` gem yet  but this is an approach others tried https://github.com/activeadmin/activeadmin/discussions/8223#discussioncomment-8274079

**Reply by matteeyah** [reply]
We're using `dartsass-rails`. I'm just trying to find a way if we can vendor the compiled CSS and use it as is. I managed to generate the raw css with `tailwindcss-rails` - https://gist.github.com/matteeyah/9913c2920283e2d8948fcd56c76a126a

It seems to be working as expected.

<img width="1552" alt="Screenshot 2024-03-01 at 19 11 32" src="https://github.com/activeadmin/activeadmin/assets/1218868/ab3b5cb5-471f-4649-ab8d-27902e6495ef">

I'm still figuring out why the JavaScript functionality seems to be broken.
